rockcraft.yaml

```diff
@@ -95,38 +95,15 @@ parts:
     build-packages:
       - wget
       - ssl-cert
+      - git
     overlay-packages:
       - python3-pip
     overlay-script: |
-      SPARK_CLIENT_PKG_VERSION='0.0.7'
       mkdir -p $CRAFT_PART_INSTALL/opt/spark/python/dist
-      cd $CRAFT_PART_INSTALL/opt/spark/python/dist
-      echo "Downloading latest available Spark wheel package release v${SPARK_CLIENT_PKG_VERSION}."
-      wget -q https://github.com/canonical/spark-client-snap/releases/download/v${SPARK_CLIENT_PKG_VERSION}/spark_client-${SPARK_CLIENT_PKG_VERSION}-py3-none-any.whl
-      if  [[ $? -ne 0 ]]
-        then
-          echo "DOWNLOAD ERROR: Spark client package v${SPARK_CLIENT_PKG_VERSION} could not be downloaded properly! Exiting...." >&2
-          exit 1
-      fi
-      pip install --target=${CRAFT_PART_INSTALL}/opt/spark/python/dist ./spark_client-${SPARK_CLIENT_PKG_VERSION}-py3-none-any.whl
+      pip install --target=${CRAFT_PART_INSTALL}/opt/spark/python/dist git+https://github.com/canonical/spark-k8s-toolkit-py.git@main
     stage:
       - opt/spark/python/dist
 
-  spark-client-apps:
-    plugin: dump
-    after: [ client-scripts ]
-    source: files/spark/bin
-    organize:
-      spark-client.pyspark: opt/spark/python/bin/spark-client.pyspark
-      spark-client.service-account-registry: opt/spark/python/bin/spark-client.service-account-registry
-      spark-client.spark-shell: opt/spark/python/bin/spark-client.spark-shell
-      spark-client.spark-submit: opt/spark/python/bin/spark-client.spark-submit
-    stage:
-      - opt/spark/python/bin/spark-client.pyspark
-      - opt/spark/python/bin/spark-client.service-account-registry
-      - opt/spark/python/bin/spark-client.spark-shell
-      - opt/spark/python/bin/spark-client.spark-submit
```

Comment:
**question** Why removing this? I believe it was sort of useful to have this short-cut

Reply:
I re-introduce them.